pages/docs/support/upgrading-grouparoo/v07-v08.mdx

```diff
@@ -0,0 +1,125 @@
+---
+title: "Upgrading Grouparoo from v0.7 to v0.8"
+date: "2022-01-06"
+pullQuote: "v0.8 introduces a new model-centric UI and brings some breaking changes.  Learn how to upgrade from v0.7 to v0.8!"
+---
+
+Grouparoo v0.8 brings a UI update focused on the Models you build. It also brings several syncing improvements, and a few breaking changes. Learn how to upgrade from v0.7 to v0.8!
+
+As always, begin by updating the packages within your Grouparoo project by [following the generic upgrade guide](/docs/support/upgrading-grouparoo#installing-updates) to bump your dependencies to `v0.8.x`.
+
+**An example of a Grouparoo deployment being upgraded [can be found here](https://github.com/grouparoo/app-example-config/pull/39).**
+
+## Model-Centric UI
+
+One major change in Grouparoo v0.8 is our new Model-Centric UI. Now, you're able to see all of the Sources, Properties, Schedules, and Destinations associated with a Model on a single dashboard. You can view a sample Record and quickly access the configuration pages for each of these objects.
+
+<Image
+  src="docs/support/upgrading-grouparoo/v07-v08-model-centric-ui.png"
+  alt="Model-Centric UI Updates"
+  width={700}
+  height={728}
+  className="mx-auto"
+/>
+
+## Airtable Plugin
```

Comment:
@bleonard Do we want to highlight other community-contributed plugins in v0.8 here?